apps/meeting/src/containers/Navigation/index.tsx

```diff
@@ -9,37 +9,90 @@ import {
   NavbarItem,
   Attendees,
   Eye,
-  SignalStrength
+  SignalStrength,
+  Flex,
+  ZoomIn,
+  ZoomOut,
+  useContentShareState,
 } from 'amazon-chime-sdk-component-library-react';
 
 import { useNavigation } from '../../providers/NavigationProvider';
 import { useAppState } from '../../providers/AppStateProvider';
-import LocalMediaStreamMetrics from '../LocalMediaStreamMetrics';
+import { LocalMediaStreamMetrics } from '../LocalMediaStreamMetrics';
+import { VideoGridMode } from '../../types';
+import GalleryLayout from '../../components/Icons/GalleryLayout';
+import FeaturedLayout from '../../components/Icons/FeaturedLayout';
+import { useVideoGridControls } from '../../providers/VideoGridProvider';
 
-const Navigation = () => {
+const Navigation: React.FC = () => {
   const { toggleRoster, closeNavbar } = useNavigation();
-  const { theme, toggleTheme } = useAppState();
+  const { theme, toggleTheme, videoGridMode, setVideoGridMode } = useAppState();
+  const { sharingAttendeeId } = useContentShareState();
+  const { zoomIn, zoomOut } = useVideoGridControls();
 
   return (
     <Navbar className="nav" flexDirection="column" container>
       <NavbarHeader title="Navigation" onClose={closeNavbar} />
-      <NavbarItem
-        icon={<Attendees />}
-        onClick={toggleRoster}
-        label="Attendees"
-      />
-      <NavbarItem
-        icon={<Eye />}
-        onClick={toggleTheme}
-        label={theme === 'light' ? 'Dark mode' : 'Light mode'}
-      />
-      <NavbarItem
-        icon={<SignalStrength />}
-        onClick={() => {}}
-        label="Media metrics"
-      >
-        <LocalMediaStreamMetrics />
-      </NavbarItem>
+      <Flex css="margin-top: 0rem;">
+        <NavbarItem
+          icon={<Attendees />}
+          onClick={toggleRoster}
+          label="Attendees"
+        />
+        <NavbarItem
```

Comment:
Nice catch. I'm actually following the Jaime's UX design. This button should be called "go to xxx mode". That's why it displayed GalleryLayout icon when it's actually Featured mode and vice verse.

I think another reason why it is confused is because the difference between two mode is not very significant, so user have to check something to tell what mode they are actually using. If you check [zoom's guide](https://support.zoom.us/hc/en-us/articles/201362323-Changing-the-video-layout-Active-Speaker-View-and-Gallery-View-#h_ac952845-ed43-4884-8263-4356f98e26f3), they use the same logic.